### PR TITLE
Centralize mime type definitions

### PR DESCRIPTION
**Pull request by AndyScherzinger**
Signed-off-by: Andy Scherzinger <info@andy-scherzinger.de>

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
Drive-by PR to tackle @AlvaroBrey's suggestion with https://github.com/nextcloud/talk-android/pull/2086#discussion_r893173055

File: app/build.gradle [hunk 1]
```diff
@@ -36,7 +36,7 @@ apply plugin: "org.jlleitschuh.gradle.ktlint"
 
 android {
     compileSdkVersion 31
-    buildToolsVersion '32.0.0'
+    buildToolsVersion '33.0.0'
```
Comment (AlvaroBrey) [inline app/build.gradle:39]: This change seems unrelated to the PR

**Review by AlvaroBrey — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/nextcloud/talk/utils/FileViewerUtils.kt [hunk 2]
```diff
@@ -249,8 +253,8 @@ class FileViewerUtils(private val context: Context, private val userEntity: User
 
     fun isSupportedForInternalViewer(mimetype: String?): Boolean {
```
[the remark this answers is not in the document]
Comment (AndyScherzinger) [reply app/src/main/java/com/nextcloud/talk/utils/FileViewerUtils.kt:254]: DONE

File: app/src/main/java/com/nextcloud/talk/utils/Mimetype.kt [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        const val TEXT_PREFIX = "text/"
+        const val AUDIO_PREFIX = "audio"
+
+        const val FOLDER = "inode/directory"
+
+        const val IMAGE_PNG = "image/png"
+        const val IMAGE_JPEG = "image/jpeg"
+        const val IMAGE_GIF = "image/gif"
+    }
+}
```
[the remark this answers is not in the document]
Comment (AndyScherzinger) [reply app/src/main/java/com/nextcloud/talk/utils/Mimetype.kt:36]: DONE

File: app/build.gradle (hunk 1 above)
Comment (AndyScherzinger) [reply app/build.gradle]: Yes, which is why I put it in a separate commit, will remove it 👍

**Review by AlvaroBrey — APPROVED** [review]
